event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/CaseUpdate.java

```diff
@@ -20,4 +21,7 @@
   private String refusalReceived;
   private CaseUpdateSample sample;
   private CaseUpdateSampleSensitive sampleSensitive;
+  private String caseRef;
+  private LocalDate createdAt;
```

Comment:
should be LocalDateTime.  
Might want to double check that the dates come out correctly after JSON conversion, or add extra test

Reply:
Tested locally and added small test to confirm outgoing JSON date is the same as incoming JSON